### ISSUE TITLE
how about using both chatgpt3.5 and chatgpt4？

### ISSUE THREAD

**Issue by shawnl111** [smol-ai/GodMode#191]
how about providing two login interfaces to log in to two accounts simultaneously, using chatgpt3.5 and chatgpt4, respectively？

**Comment by swyxio** [comment]
yep we thought about it! past attempt here https://github.com/smol-ai/GodMode/pull/70

 right now each provider is implemented as a singleton (hence we do the lookup for element id) and we'd need to rewrite the providers to do this properly. its not impossible, just mildly annoying to rewrite. if you're up for the challenge, i'd happily take a PR :)
